package.json

```diff
@@ -40,17 +40,20 @@
     "@bower_components/bootstrap4": "twbs/bootstrap#v4.0.0",
     "@bower_components/ng-ckeditor": "esvit/ng-ckeditor#^0.2.1",
     "@ovh-ux/manager-banner": "^1.1.0",
+    "@ovh-ux/manager-cloud-styles": "^0.3.0",
     "@ovh-ux/manager-config": "^0.3.0",
     "@ovh-ux/manager-core": "^7.1.2",
+    "@ovh-ux/manager-enterprise-cloud-database": "git+https://github.com/ovh-ux/manager.git#feat/enterprise-cloud-database-split",
     "@ovh-ux/manager-navbar": "^1.2.2",
-    "@ovh-ux/manager-server-sidebar": "^0.5.2",
+    "@ovh-ux/manager-server-sidebar": "git+https://github.com/ovh-ux/manager.git#feat/enterprise-cloud-database-sidebar-split",
```

Comment:
This version will be updated once https://github.com/ovh-ux/manager/pull/1242 is merged